app/src/main/resources/common/README.md.mustache

```diff
@@ -121,12 +137,36 @@ Or via:
 java -Xdebug -Xrunjdwp:transport=dt_socket,address=5000,server=y,suspend=y -jar build/libs/{{appName}}.war
 ```
 
-Run the CAS web application as a *standalone* executable WAR:
+## Build and run CAS as a *standalone* executable WAR
+
+This allows you to run the CAS web application as a *standalone* executable WAR on Linux and Mac.
+
+The build system will add a startup script inside the war, that makes it executable.
```

Comment:
This is not something we can add; it's not maintainable if it ever changes since the behavior is controlled by an external library and they could change this at any moment.